proto/fundraising/fundraising.proto

```diff
@@ -222,7 +225,7 @@ message AllowedBidder {
   string bidder = 1;
 
   // max_bid_amount specifies the maximum bid amount that the bidder can bid
-  string max_bid_amount = 2 [
+  string max_bid_amount = 3 [
```

Comment:
It seems this must be 2. Or is there other parameter added to `AllowedBidder`?

Reply:
Thanks! Yes, it must be 2.